ArmstrongPy.py

```diff
@@ -1,13 +1,16 @@
 # Python program to check if the number is an Armstrong number or not
-num = int(input("Enter a number: "))
-sum = 0
-
-temp = num
-while temp > 0:
- digit = temp % 10
- sum += digit ** 3
- temp //= 10
-if num == sum:
- print(num,"is an Armstrong number")
-else:
- print(num,"is not an Armstrong number")
\ No newline at end of file
+class ArmStrong():
+ def Armstrong(self):
+    num = int(input("Enter a number: "))
```

Comment:
Please pass the input directly, don't use the input to read it from console

Reply:
Noted and made the required changes, kindly verify  it

Reply:
Resolve the conflicts

Reply:
Conflicts are resolved, Kindly check